app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class OrganisationsController < ApplicationController
+  before_action :set_organisation, only: %i[edit update]
+  before_action :validate_user_is_part_of_organisation, only: %i[edit update]
+
+  def index; end
+
+  def edit; end
+
+  def update
+    if @organisation.update(organisation_params)
+      redirect_to organisation_path(@organisation)
+      flash[:notice] = 'Organisation updated'
+    else
+      render :edit
+    end
+  end
+
+private
+
+  def set_organisation
+    @organisation = Organisation.find(params.fetch(:id))
```

Comment:
We do have a helper method called
```current_organisation```

So this could just be 
```
@organisation = current_organisation
```

Reply:
I think its done in this this way to ensure not anyone can just hardcode the URL and change another organisations settings. @emileswarts

Reply:
From my understanding now:

let's say I'm organisation ID 1
if I go to `organisations/1` then that's the same as `current_organisation`
but if I go to `organisations/2` then that's `Organisation.find(2)`, which isn't my `current_organisation`
so the two aren't equivalent